src/index.js.flow

```diff
@@ -1,6 +1,6 @@
-/* flow strict */
+ /* @flow strict */
 
-declare module 'time-elements' {
+declare module '@github/time-elements' {
```

Comment:
Can we drop `declare module` or at least make it anonymous so we don't have to keep updating it through renames?

Reply:
Yup. I'm linking it to `github/github` right now to make sure it works with the dropped `module` scope.

Reply:
We talked in slack but for posterity, we need this to be declared in a module since we include it in the `[libs]` section of our `.flowconfig` and we need to signal to `flow` what module this file is for.